images/atomic-openshift-node-problem-detector.yml

```diff
@@ -0,0 +1,30 @@
+container_yaml:
+  go:
+    modules:
+    - module: github.com/openshift/node-problem-detector
+content:
+  source:
+    dockerfile: Dockerfile.openshift.rhel7
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/node-problem-detector.git
```

Comment:
... we don't want to add this in, it's 4.6-only.

Reply:
yeah, it's weird, I rebased then somehow it's included.